docs/_future/getting_started_future.md

```diff
@@ -0,0 +1,146 @@
+
+---
+
+### Setting up {#setting-up}
+
+#### Slack CLI {#setting-up-cli}
+
+To build a next-generation app with Bolt for Python, you'll need to get the Slack CLI.
+
+Install the Slack CLI by following this <a href="https://api.slack.com/future/quickstart" target="_blank">Quickstart</a>. Since we won't be using Deno to build our next-generation app, you can skip any instructions related to installing Deno or creating an app using a Deno template. Once you've logged into the CLI using `slack login` and verified your login using `slack auth list`, you can proceed with the instructions in this guide.
+
+#### Dependencies {#setting-up-dependencies}
+
+Once the CLI is set up, make sure your machine has the most recent version of <a href="https://www.python.org/" target="_blank">Python</a> installed. You can install Python through a package manager (such as <a href="https://brew.sh/" target="_blank">Homebrew</a> for macOS) or directly from the <a href="https://www.python.org/downloads/" target="_blank">website</a>.
+
+---
+
+### Create a new app {#create-app}
+
+Before you start developing with Bolt, you'll want to create a Slack app.
+
+To create the app, you'll run the following command:
+
+```bash
+slack create my-app -t slack-samples/bolt-python-starter-template -b future
+```
+
+This command creates an app through the CLI by cloning a specified template. In this case, the template is the <a href="https://github.com/slack-samples/bolt-python-starter-template/tree/future" target="_blank">Bolt for Python Starter Template</a> on the `future` branch. This starter template includes a "Hello World" example that demonstrates how to use <a href="/bolt-python/future/concepts#built-in-functions" target="_blank">built-in</a> and <a href="/bolt-python/future/concepts#functions" target="_blank">custom</a> Functions, <a href="https://api.slack.com/future/triggers" target="_blank">Triggers</a> and <a href="/bolt-python/future/concepts#manifest-workflows" target="_blank">Workflows</a>.
+
+Once the app is successfully created, you should see a message like this:
+
+```text
+✨ my-app successfully created
+
+🧭 Explore your project's README.md for documentation and code samples, and at any time run slack help to display a list of available commands
+
+🧑‍🚀 Follow the steps below to try out your new project
+
+1️⃣  Change into your project directory with: cd my-app
+
+2️⃣  Develop locally and see changes in real-time with: slack run
+
+3️⃣  When you're ready to deploy for production use: slack deploy
+
+🔔 If you leave the workspace, you won’t be able to manage any apps you’ve deployed to it. Apps you deploy will belong to the workspace even if you leave the workspace
+```
+
+---
+
+### Set up your trigger {#setup-trigger}
+
+As mentioned, this app comes with pre-existing functionality - it uses Functions, Workflows and a Trigger set up to kick things off. Let's run a command to initialize that Trigger via the CLI.
```

Comment:
Let's link the capital letter terms to the relevant document sections!

I think we should also explain _why_ the user would want to initialize a Trigger, otherwise we're not teaching developers what this concept is or why they should care about them. Maybe something like:

```suggestion
As mentioned, this app comes with pre-existing functionality - it uses Functions, Workflows and a Link Trigger (link to link trigger docs: https://api.slack.com/future/triggers/link) that will allow users in Slack to initiate the functionality provided by the app. Let's run a command to initialize that Link Trigger via the CLI.
```